### PR TITLE
fix: App goes to crash in /users page

### PR DESCRIPTION
**Pull request by Roman-Kholopov**

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ class User {
   async fetchUsers() {
     try {
       const response = await axios.get('https://jsonplaceholder.typicode.com/users');
+      // const response = await axios.get('https://jsonplaceholder.tode.com/users');
```
Comment (naHDop-tech) [inline models/User.js]: delete if don't need

File: controllers/user.js [hunk 2]
```diff
@@ -1,17 +1,36 @@
 const User = require('../models/User');
 
-async function getUserList(req, res) {
-    const user = new User();
+// async function getUserList(req, res) {
```
Comment (naHDop-tech) [inline controllers/user.js]: delete all comments

**Comment by naHDop-tech** [comment]
2 comment and LGTM

File: models/User.js (hunk 1 above)
Comment (Roman-Kholopov) [reply models/User.js]: deleted

File: controllers/user.js (hunk 2 above)
Comment (Roman-Kholopov) [reply controllers/user.js]: deleted

**Review by naHDop-tech — APPROVED** [review]

**Pull request merged**